staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion_test.go

```diff
@@ -406,26 +406,30 @@ func failureResponseConverter(message string) func(review apiextensionsv1beta1.C
 func nontrivialConverter(desiredAPIVersion string, obj runtime.RawExtension) (runtime.RawExtension, error) {
 	u := &unstructured.Unstructured{Object: map[string]interface{}{}}
 	if err := json.Unmarshal(obj.Raw, u); err != nil {
-		return runtime.RawExtension{}, fmt.Errorf("Fail to deserialize object: %s with error: %v", string(obj.Raw), err)
+		return runtime.RawExtension{}, fmt.Errorf("failed to deserialize object: %s with error: %v", string(obj.Raw), err)
 	}
 
-	currentAPIVersion := u.Object["apiVersion"]
-	if currentAPIVersion == "v1beta2" && desiredAPIVersion == "v1beta1" {
+	currentAPIVersion := u.GetAPIVersion()
+
+	fmt.Printf("Converting %s/%s from %s to %s\n", u.GetNamespace(), u.GetName(), currentAPIVersion, desiredAPIVersion)
```

Comment:
drop debug printing?

Reply:
wiring t there is awkward (it changes due to subtests). Will remove it.